resources/windows/wix/service.pxp-agent.wxs.erb

```diff
@@ -33,10 +33,10 @@
           <RegistryValue Name="AppParameters"
                          Value=""
                          Type="expandable" />
-          <RegistryValue Name="AppEnvironmentExtra"
-                         Value="PATH=[INSTALLDIR]pxp-agent\bin;[INSTALLDIR]puppet\bin;[INSTALLDIR]sys\ruby\bin;%PATH%"
-                         Type="multiString"
-                         Action="append" />
+          <RegistryValue Name="AppEnvironmentExtra" Type="multiString">
+                         <MultiStringValue>PATH=[INSTALLDIR]pxp-agent\bin;[INSTALLDIR]puppet\bin;[INSTALLDIR]facter\bin;[INSTALLDIR]sys\ruby\bin;%PATH%</MultiStringValue>
```

Comment:
So Wix docs say that you can have `0..unbounded` for `MultiStringValue` - http://wixtoolset.org/documentation/manual/v3/xsd/wix/registryvalue.html

And additionally

> Please note that this value will only result in a multi-string value if there is more than one registry value or the Action attribute's value is 'append' or 'prepend'. Otherwise a string value will be created.


Have you tested this though to make sure it does what is desired?  Did you get these values by using NSSM locally and exporting the reg keys?

Reply:
I did test it out and it seemed to work. I didn't try this using NSSM locally, hand-wrote it.

Seems to follow the docs you quoted.

Reply:
Could someone from @puppetlabs/windows help with the validation here?

Reply:
Also, hosts `windows-1` and `windows-2` on orch-next are using that build.

Reply:
Also need to test uninstall to make sure it removes itself cleanly.

Reply:
I'll test out uninstall.

Reply:
Uninstall successfully removes the entire services/pxp-agent key section.

Reply:
Checking out upgrades as well now.

Reply:
Upgrades look good too.